src/Loader/LoaderInterface.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Tuf\Loader;
+
+use Psr\Http\Message\StreamInterface;
+
+/**
+ * Defines an interface to load data as a stream.
+ *
+ * The data source can be anything, from anywhere, but it must be returned as an
+ * implementation of \Psr\Http\Message\StreamInterface.
+ *
+ * This is an internal interface used to load untrusted data! External code
```

Comment:
Right now, there's no real reason for anything external to TUF to implement this interface. There may be in the future, but for now it makes sense for it to be internal. It needs to exist as an interface in order for the decorator pattern to work properly.

Reply:
I am not sure that Updater should implement this interface. Does `\Tuf\Client\Updater::load` return untrusted data?

Reply:
That's a good point. Removed the interface from Updater.